RecoTracker/IterativeTracking/python/iterativeTk_cff.py

```diff
@@ -24,17 +24,25 @@
 
 import RecoTracker.IterativeTracking.iterativeTkConfig as _cfg
 
-iterTrackingEarly = _cfg.createEarlySequence("", "", globals())
+#iterTrackingEarly = _cfg.createEarlySequence("", "", globals())
+#for _eraName, _postfix, _era in _cfg.nonDefaultEras():
+#    _era.toReplaceWith(iterTrackingEarly, _cfg.createEarlySequence(_eraName, _postfix, globals()))
+#
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
No. I left it in to show what I replaced.

Reply:
Our coding style does not imply that we leave previous implementations in place.